model/shared/src/main/scala/explore/model/display.scala

```diff
@@ -76,4 +79,18 @@ object display {
     s"${cs.imageQuality.label} ${cs.cloudExtinction.label} ${cs.skyBackground.label}$wv$er"
   }
 
+  // Not implicit. When we have opaque types we may define a BrightnessValue.
+  // Scientific notation only kicks in when more than 3 decimal digits would be needed.
+  val displayBrightness: Display[BigDecimal] = Display.byShortName { x =>
+    if (x.scale <= 3 && x.precision <= 5)
+      x.toString
+    else {
+      // TODO The same scientific notation formatting is also defined in lucuma-ui.
+      // Should we move it to a Format[String, BigDecimal] in lucuma-core?
+      val formatter = new DecimalFormat("0.0E0")
```

Comment:
are you using the formatter from `scala-java-locales`?

Reply:
Yes